packages/subscription-manager/src/types.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module 'calypso/state/initial-state';
+declare module 'calypso/state/persisted-state';
+declare module '@automattic/wpcom-template-parts' {
+	export interface HeaderProps {
+		className?: string;
+		isLoggedIn: boolean;
+		sectionName?: string;
+		logoColor?: string;
+		variant?: 'default' | 'minimal';
+	}
+
+	export function UniversalNavbarHeader( props: HeaderProps ): JSX.Element;
```

Comment:
> If there's an issue with internal type resolution like this, it normally indicates a deeper issue. If I had to guess, it's because packages/subscription-manager/package.json should declare every dependency it uses, even internal ones. That lets the compiler + package manager resolve all the types correctly :) So that probably needs to reference wpcom-template-parts as a yarn workspace dependency

Spot on. Thanks for noticing this one. 🙇‍♂️